content/en/docs/_index.md

```diff
@@ -8,12 +8,12 @@ draft: false
 Welcome to the {{< mtf-wiki >}}, a place that may help some transgender women.
 
 {{< notice info >}}
-Thought this site was founded initially for Chinese trans, we provide this English page to show our visibility. However, if you would like to contribute information for trans in your region, welcome to do it.
+Though this site was founded initially for Chinese trans, we provide this English page to show our visibility. However, if you would like to contribute information for trans in your region, welcome to do it.
 {{< /notice >}}
 
-MtF (Male-to-Female), transgender female, refers to a female whose sex[^1] is male when she was born.
+MtF (Male-to-Female), or transgender female, refers to a female whose gender[^1] was assigned male at birth but is a woman.
 
-{{< mtf-wiki >}} is committed to becoming a free, open and continuously updated transgender knowledge integration site, where you can learn about all kinds of medical information and dailylife guides related to transgender.
+{{< mtf-wiki >}} is dedicated to evolving into a free, open, and consistently updated platform for the integration of transgender knowledge, offering insights into various medical information and daily life guides relevant to transgender individuals and their experiences.
```

Comment:
Mtf.wiki is committed to becoming a free, open, and constantly updated platform for the integration of transgender knowledge, providing insights into diverse medical information and everyday living guidelines important to transgender individuals and their experiences.